### PR TITLE
Add type annotations to controllers/multizone.py

### PR DESCRIPTION
**Pull request by emontnemery**

### CODE REVIEW

File: pychromecast/controllers/multizone.py [hunk 1]
```diff
@@ -152,50 +210,36 @@ def register_listener(self, member_uuid, listener: MultiZoneManagerListener):
...
-        """The cast has been added to group identified by group_uuid."""
-
-    @abc.abstractmethod
-    def multizone_member_removed(self, group_uuid: str):
-        """The cast has been removed from group identified by group_uuid."""
-
-    @abc.abstractmethod
-    def multizone_status_received(self):
-        """Multizone status has been updated."""
+        return self._groups[str(group_uuid)]["chromecast"].media_controller  # type: ignore[no-any-return]
```
Comment (emontnemery) [inline pychromecast/controllers/multizone.py:230]: The mypy ignore will be removed in an upcoming PR which adds type annotations to the `Chromecast` class

File: pychromecast/controllers/multizone.py [hunk 2]
```diff
@@ -41,106 +114,91 @@ def __init__(self, group_cast, casts):
...
+        self._groups: dict[str, GroupInfo] = {}
 
-    def add_multizone(self, group_cast):
+    def add_multizone(self, group_cast: Chromecast) -> None:
         """Start managing a group"""
         self._groups[str(group_cast.uuid)] = {
             "chromecast": group_cast,
             "listener": Listener(group_cast, self._casts),
-            "members": set(),
+            # "members": set(),
```
Comment (MartinHjelmare) [inline pychromecast/controllers/multizone.py:186]: Commented code.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: pychromecast/controllers/multizone.py (hunk 2 above)
Comment (emontnemery) [inline pychromecast/controllers/multizone.py:186]:
```suggestion
```

Comment (emontnemery) [reply pychromecast/controllers/multizone.py]: Sorry about that, removed now

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**